gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1036,6 +1026,101 @@ double ofx_get_investment_amount(const struct OfxTransactionData* data)
     }
 }
 
+// Forward declaration, required because several static functions depend on one-another.
+static void
+gnc_file_ofx_import_int (ofx_info* info);
```

Comment:
If you need this I think it should be at the start and I not sire on function name, why not something like gnc_file_ofx_import_process_file

Reply:
I'll move to the head. I was told that aux functions typically have an _int suffix in GC (which seems strange to me). I'm all for changing it to gnc_file_ofx_import_process_file

Reply:
I think you may be referring to the file changes, probably not the right words, in that case there were to two functions declared in the h file that really just provided the right parameters to the common _int function.

Reply:
Yes, that's it. I'll change the name then.